docs/csharp/programming-guide/concepts/linq/how-to-add-custom-methods-for-linq-queries.md

```diff
@@ -20,26 +20,28 @@ public static class LINQExtension
 {
     public static double Median(this IEnumerable<double> source)
     {
-        if (source.Count() == 0)
+        var countOfElementsInTheSet = source?.Count();
```

Comment:
This will make `countOfElementsInTheSet` an `int?`.  Let's change it to this:

```suggestion
        var countOfElementsInTheSet = source?.Count() ?? 0;
```